resources/js/electron-plugin/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@nativephp/electron-plugin",
+  "version": "tarball",
```

Comment:
This might seem weird. I removed the version at first, but `npm pack` requires a package name and version.
the created taball then includes that version in the filename: `nativephp-electron-plugin-tarball.tgz`.

In our case we need the tarball to have the same name, hence this string. The filename of the tarball is referenced in the Build command here: https://github.com/NativePHP/electron/blob/761d10e1ff5a718f08a74aa7d50331a98f077fd2/src/Commands/BuildCommand.php#L46

Reply:
If we randomised the version at runtime and injected this into the package.json, would that negate the need for `ci`?

Does the version need to be anything specific (incrementing/increasing?) or can it just be random?

Reply:
The serve script does do a clean install for everything. I'm not super happy with that yet.

Reply:
No, that feels very wasteful